packages/block-library/src/comment-template/edit.js

```diff
@@ -25,9 +30,63 @@ const TEMPLATE = [
 
 function CommentTemplateInnerBlocks() {
 	const innerBlocksProps = useInnerBlocksProps( {}, { template: TEMPLATE } );
-	return <li { ...innerBlocksProps } />;
+	return <li { ...innerBlocksProps }></li>;
 }
 
+/**
+ * Component that renders a list of (nested) comments. It is called recursively
+ * in its own body.
+ *
+ * @param {Object} props                         Component props.
+ * @param {Array}  [props.blockProps]            - Props from parent's `useBlockProps()`.
+ * @param {Array}  [props.blockContexts]         - Array of comment objects.
+ * @param {Array}  [props.activeBlockContext]    - The block that is currently active.
+ * @param {Array}  [props.setActiveBlockContext] - The setter for activeBlockContext.
+ * @param {Array}  [props.blocks]                - Array of blocks returned from
+ *                                               getBlocks() in parent .
+ * @return {WPElement}                 		List of comments.
+ */
+const RenderComments = ( { blockContexts, ...props } ) => (
+	<ol { ...props.blockProps }>
```

Comment:
It will become an issue as soon as we start adding block supports. There should be always only a single block wrapper. Let's make sure those props don't get propagated.

Reply:
gotcha, thanks Greg